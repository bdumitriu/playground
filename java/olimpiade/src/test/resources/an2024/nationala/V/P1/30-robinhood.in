3
633
332 331
