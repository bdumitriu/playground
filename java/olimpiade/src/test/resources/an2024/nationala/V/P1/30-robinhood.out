5
5 9 13 17 21 25 29 33 37 41 45 49 53 57 61 65 69 73 77 81 85 89 93 97 101 105 109 113 117 121 125 129 133 137 141 145 149 153 157 161 165 169 173 177 181 185 189 193 197 201 205 209 213 217 221 225 229 233 237 241 245 249 253 257 261 265 269 273 277 281 285 289 293 297 301 305 309 313 317 321 325 329 337 341 345 349 353 357 361 365 369 373 377 381 385 389 393 397 401 405 409 413 417 421 425 429 433 437 441 445 449 453 457 461 465 469 473 477 481 485 489 493 497 501 505 509 513 517 521 525 529 533 537 541 545 549 553 557 561 565 569 573 577 581 585 589 593 597 605 609 613 617 621 625 629 
